shared/test/test_net_sim_api.rb

```diff
@@ -0,0 +1,55 @@
+require 'minitest/autorun'
+require 'rack/test'
+require File.expand_path '../../../deployment', __FILE__
+require File.expand_path '../../middleware/net_sim_api', __FILE__
+
+ENV['RACK_ENV'] = 'test'
```

Comment:
This _does_ make real API calls.  In my local environment this hits my local MySQL-backed channels API, and on test it should hit the test-specific DynamoDB-backed channels API.  If we're uncomfortable with that we should change this and the tables API tests in test_tables.rb.